service/src/main/java/io/camunda/service/AuthorizationServices.java

```diff
@@ -37,7 +37,7 @@ public AuthorizationServices(
       final CamundaSearchClient searchClient,
       final ServiceTransformers transformers,
       final Authentication authentication) {
-    super(brokerClient, searchClient, transformers, authentication);
+    super(brokerClient, searchClient, transformers, authentication, false);
```

Comment:
💭 Authorizations are a tricky one, in one instance the authorizations needs to be retrieved without checks so they can be applied effectively but in the other instance when I want to view authorizations via REST then I should only see them if I am authorized to, I think this is likely something that we need to address in an alternate way, perhaps with a different service class or handler

Reply:
Yes, good point. 
I believe we need to differentiate between “business” resources (such as processes and decisions) and infrastructure resources (like authorizations, users, and groups). 
While infrastructure resources should be accessible to the applications, we need to apply the authorization checks when these resources are exposed through the API.

Reply:
The differentiation sounds fair to me there, I guess really its down to the context i.e. when I get the authorizations on an infra level I don't want the checks, but when its at a business level then I do. That feels like something that we can just handle as a different service I think so the service for the business level stuff has this enabled, but the other service for internal infra stuff does not

Reply:
The boolean flag might become a potential source of issues. When do I set to `true` when to `false`, does the service that I have right now does authorization checks or not, etc.

Can't we make it more explicit? [For example in C7](https://github.com/camunda/camunda-bpm-platform/blob/64274723bb94a91343e0acba1b08920459439f6c/engine/src/main/java/org/camunda/bpm/engine/impl/interceptor/CommandContext.java#L540), there is a method called `runWithoutAuthorization()`which must be explicitly used to execute something without any authorization check. That way, it is also explicit in the code of a service and clear what is happening.